### PR TITLE
Print slug size before upload in packer's atlas post-processor

### PR DESCRIPTION
**Pull request by grubernaut**
Adds size output to `ui.Message` as well as if the artifact failed to upload to atlas.

### CODE REVIEW

File: post-processor/atlas/post-processor.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ func (p *PostProcessor) PostProcess(ui packer.Ui, artifact packer.Artifact) (pac
 		opts.FileSize = r.Size
 	}
 
-	ui.Message("Uploading artifact version...")
+	ui.Message(fmt.Sprintf("Uploading artifact (Size: %v)", opts.FileSize))
```
Comment (cbednarski) [inline post-processor/atlas/post-processor.go]: Can we change this to the unit of measure? E.g. `%d bytes`?

**Comment by grubernaut** [comment]
Absolutely! 
Just realised, `opts.FileSize` is the length in bytes of the file, might make more sense to display this in `MB` instead of `bytes`? 
https://github.com/hashicorp/atlas-go/blob/master/archive/archive.go#L112
https://golang.org/pkg/os/#FileInfo

**Comment by grubernaut** [comment]
This will, however, report `0.00MB` for amazon.ami artifact types during packer builds of ami's.

**Comment by cbednarski** [comment]
@grubernaut Thanks! I'm going to merge this. Bytes is preferable for debugging, since it can easily be compared with output from other tools (e.g. `ls -l`), does not lose precision, and avoids the 1000 vs. 1024 issue.

**Pull request merged**